docs/notebooks/documents/getting_started.py

```diff
@@ -0,0 +1,159 @@
+# Otherwise, you may use the sections of this notebook to create or import signing keys for
+# transactions, configure PDO services (e.g. by importing a PDO `site.toml` file), and configure
+# service groups to simplify contract creation.
+#
+# %% [markdown]
+# ## Initialize the Interpreter
+#
+# The following cell loads modules that configure the Jupyter interpreter and simplify interaction
+# with the PDO environment.
+# %%
+import os
+import pdo.contracts.jupyter as pc_jupyter
+import IPython.display as ip_display
+import ipywidgets
+
+pc_jupyter.load_ipython_extension(get_ipython())
+
+try : state
+except NameError:
+    (state, bindings) = pc_jupyter.initialize_environment('unknown')
+
+# %% [markdown]
+# ## Create Keys
+#
+# PDO uses ECDSA cryptographic keys to sign transactions. Each key pair represents an identity or
+# role. The list below will show a list of currently available keys. There is a form to use to
+# create new keys (the refresh button will update the key list when you create new keys).
+# Additional management of keys is provided by the [Key Manager Notebook](key_manager.ipynb).
+#
+# %%
+from pdo.contracts.keys import PrivateKeyListWidget, GenerateKeyWidget
+
+key_list = PrivateKeyListWidget(state, bindings)
+key_gen = GenerateKeyWidget(state, bindings)
+ip_display.display(ipywidgets.VBox([key_list, key_gen]))
+
+# %% [markdown]
+# ## Import Site Information
+#
+# PDO clients require information about the services that will be used for contracts. Typically, a
+# service provider will create a site description file (often called `site.toml`) that can be used
+# to import information about a number of services.
+#
+# The widget below contains information about the currently created service groups and a form
+# for uploading a new site file. Additional management of service information is provided by the
+# [Service Manager Notebook](service_manager.ipynb)
+#
+# %%
+from pdo.contracts.services import ServiceListWidget
+from pdo.contracts.services import ServiceUploadWidget
+from pdo.contracts.services import service_labels
+
+children = map(lambda stype : ServiceListWidget(state, bindings, stype), service_labels.keys())
+service_list = ipywidgets.Tab(children=list(children), titles=list(service_labels.values()))
+ip_display.display(ipywidgets.VBox([service_list, ServiceUploadWidget(state, bindings)]))
+
+# %% [markdown]
+# ## Create Service Groups
+#
+# Service groups are a shortcut for configuration of collections of enclave, storage, and
+# provisioning services used to create and provision a PDO contract object. Each type of
+# service will have its own service groups. PDO requires that each type of service have a
+# service group named "default" that will be used when no groups have been specified.
+#
+# The widget below contains a list of service groups for each type of service and a widget for
+# creating new services of each type. If there is no default service group, please create one.
```

Comment:
Is there a way to link to `service_groups_manager.py` from here?

Reply:
Yes. thought i did. will fix.